docs/api/log.md

```diff
@@ -0,0 +1,17 @@
+---
+id: log
+title: log
+---
```

Comment:
No need to have a separate file for log at the moment. Let's keep it in the `utils` namespace only.